assets/src/utils/helpers.ts

```diff
@@ -0,0 +1,14 @@
+// parse value and return accurate boolean
+// returns boolean values for both numbers and strings
+export const parseBool = (val: any) => {
```

Comment:
we already have something here https://github.com/Simon-Initiative/oli-torus/blob/master/assets/src/utils/common.ts#L49 perhaps there is a new case?